code/modules/migrations/mysql/mysql_controller.dm

```diff
@@ -0,0 +1,57 @@
+GLOBAL_DATUM(migration_controller_mysql, /datum/migration_controller/mysql)
+
+/datum/migration_controller/mysql
```

Comment:
[Please document your code to meet dmdoc requirements.](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md)